travis/setup.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
 set -ex
 
-./tools/dart/update.py
+export CLOUDSDK_CORE_DISABLE_PROMPTS=1
+curl https://sdk.cloud.google.com | bash
```

Comment:
oh right, I added that when trying to figure out what "download_from_google_storage" was, but that turns out to be in depot_tools. Let me try removing it before I check it in...